usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+
+SWAP_FILE = "/swap"
+
+
+# Workaround for https://github.com/OpenSCAP/openscap/issues/1796
```

Comment:
This is really to avoid the OOM killer, we should add that as comment rather than the reference to an issue where the code reader has to fish through lengthy comments to understand why we are setting up swap.

Reply:
Fixed.